index.js

```diff
@@ -84,6 +84,11 @@ function assembleStyles() {
 				open: `\u001B[${style[0]}m`,
 				close: `\u001B[${style[1]}m`
 			};
+
+			Object.defineProperty(styles[styleName], 'data', {
+				value: style,
+				enumerable: false
+			});
```

Comment:
oh sry, that is anthor patch when wanna use raw data, without parse style.open, style.close back to color code